src/components/TmBlock/TmBlock.vue

```diff
@@ -57,6 +56,10 @@ export default {
     TmLiTransaction
   },
   props: {
+    currentUser: {
```

Comment:
i didn't know you could make props objects with types!

Reply:
omg js we should really be doing it all over the place with default values etc 😅

Reply:
Yes this is a good idea 👍